app/models/comment.rb

```diff
@@ -206,15 +206,15 @@ def unflag_comment
   end
 
   def liked_by(user_id)
-    likes.where(user_id: user_id).count > 0
+    likes.where(user_id: user_id).size > 0
```

Comment:
@Jolios lets refactor this further so as to remove the codeclimate warning...lets change this to `likes.where(user_id: user_id).present?` 
Then we should be good..Thanks

Reply:
Hi @cesswairimu , I did as you asked and the codeclimate warning is gone  but the Travs CI build fails. It seems `@related_tags` is not defined in tags controller

Reply:
No worries let me try restarting travis since its unrelated..Thanks for the fix :100:

Reply:
The issue is solved now...I will try to restart travis